iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -216,6 +220,7 @@ def _adjust_operator_bundle(manifests_path, metadata_path, organization=None, pi
     organization_customizations = conf['iib_organization_customizations'].get(organization, [])
```

Comment:
If the defined organization_customization does not specify `related_bundles`, then the related bundles file will not be generated, right? I think that's ok, but we need to document this. In the README.md?